company-clang.el

```diff
@@ -109,8 +179,9 @@ or automatically through a custom `company-clang-prefix-guesser'."
 
 ;; TODO: Handle Pattern (syntactic hints would be neat).
 ;; Do we ever see OVERLOAD (or OVERRIDE)?
+;; Parse completion's comments/documentation too
 (defconst company-clang--completion-pattern
-  "^COMPLETION: \\_<\\(%s[a-zA-Z0-9_:]*\\)\\(?: : \\(.*\\)$\\)?$")
+   "^COMPLETION: \\_<\\(%s[a-zA-Z0-9_:]*\\) : \\(.*?\\)\\(?: : \\(.*\\)\\)\\{,1\\}$")
```

Comment:
I've changed the regexp:
"^COMPLETION: \\_<\\(%s[a-zA-Z0-9_:]*\\)\\(?: : \\(.+?\\)?\\)?\\(?: : \\(.+?\\)?\\)?$"

Prefix: **foo**

STRING | GROUP CAPTURED
--------|--------
"COMPLETION: " | / (no prefix)
"COMPLETION: blabla" | / (wrong prefix)
"COMPLETION: foobar" | 1 (2, 3 are nil)
"COMPLETION: foobar : " | 1 (2, 3 are nil)
"COMPLETION: foobar : [#int#]foobar(<#int a#>)" | 1, 2 (3 is nil)
"COMPLETION: foobar : [#int#]foobar(<#int a#>) : " | 1, 2 (3 is nil)
"COMPLETION: foobar : [#int#]foobar(<#int a#>) : Comment 1" | 1, 2, 3
"COMPLETION: foobar : [#int#]foobar(<#int a#>) : Comment 1 : " | 1, 2, 3
"COMPLETION: foobar : [#int#]foobar(<#int a#>) : Comment 1 : xyz" | 1, 2, 3